x/twap/listeners_test.go

```diff
@@ -11,14 +11,14 @@ import (
 func (s *TestSuite) TestCreateTwoAssetPoolFlow() {
 	poolId := s.PrepareUni2PoolWithAssets(defaultUniV2Coins[0], defaultUniV2Coins[1])
 
-	expectedTwap, err := types.NewTwapRecord(s.App.GAMMKeeper, s.Ctx, poolId, denom0, denom1)
+	expectedTwap, err := types.NewTwapRecord(s.App.GAMMKeeper, s.Ctx, poolId, denom1, denom0)
```

Comment:
What's the context behind all these denom order changes?

Reply:
Second this. Wish there was a way we could make `Asset0Denom: denom0` to prevent mental gymnastics that may need to occur when working on this in the future.

Reply:
I think its because of how the twap assets are alphabetically ordered (maybe)

Reply:
reverted